### DISCUSSION TITLE
Update in Fips provider from 3.0.8

### DISCUSSION THREAD

**Discussion by hkumar0520** [openssl/openssl#24886] [Q&A]
Hi Team,

Currently, we are using OpenSSL 3.1.2 and in it using OpenSSL 3.0.8 fips provider. We updated openssl 3.1.2 to 3.3.1. 
For the Fips provider, can we use a 3.0.8 or later version without/with fewer security issues observed so far?
Could you please advise on the latest version?

**Comment by paulidale** [comment]
You can continue to use the 3.0.8 FIPS provider.  Unless you use XTS mode on an arch/64 platform, it's the latest.

**Reply by hkumar0520** [reply]
Our application will support Windows 2016, 2019, and 2022. If I have to upgrade, what is the latest provider version?

**Reply by paulidale** [reply]
3.0.9 is also validated.

**Reply by hkumar0520** [reply]
We've updated to OpenSSL 3.3.1 and are using the FIPS provider version 3.0.9. However, the fix CVE-2024-4603 isn't available in 3.0.9. Should we replace the fix commit files( https://github.com/openssl/openssl/commit/3559e868e58005d15c6013a0c1fd832e51c73397), or is there another FIPS-validated version we can use as per the portal the last fips validated version is 3.0.9?

**Reply by paulidale** [reply]
There is no later FIPS validated version currently.  You'll have to use the workaround mentioned on the [FIPS and CVE page](https://openssl-library.org/news/fips-cve/index.html).

Any change to the FIPS boundary, no matter how trivial, negates the validation.
